include/envoy/server/admin.h

```diff
@@ -15,6 +15,7 @@
 namespace Envoy {
 namespace Server {
 
+class AdminFilter;
```

Comment:
nit: I'm not in favor of exposing out the AdminFilter object in this manner into the abstract includes. Can we add an `AdminFilter` or `AdminStream` interface here and then fill it in with relevant callbacks? Then the existing admin filter can implement this interface as needed?

Reply:
Either is fine with me.

Reply:
@mattklein123 @jmarantz  - I pushed a fix. Not sure if what I did is what you meant here. Please review.